src/main/java/com/neatresults/mgnltweaks/json/JsonBuilder.java

```diff
@@ -431,6 +438,59 @@ public Object remove(final Object key) {
             return obj;
         }
 
+		@Override
+		public Object get(Object key) {
+			Object superResult = super.get(key);
+			if (!(superResult instanceof ContentMap))
+				return superResult;
+
+			Node node = ((ContentMap) superResult).getJCRNode();
+			if (isArrayParent(node))
+                return childrenAsContentMapList(node);
+
+            return superResult;
+		}
+
+        private boolean isArrayParent(Node candidate) {
+            for (String key : config.childrenArrayCandidates.keySet()) {
+                final String value = config.childrenArrayCandidates.get(key);
+                final Pattern valuePattern = Pattern.compile(value);
+                final Pattern keyPattern = Pattern.compile(key);
+
+                try {
+                    if (!asPropertyStream(candidate.getProperties()).filter(
+							p -> {
+								try {
+									return keyPattern.matcher(p.getName()).matches()
```

Comment:
try/catch inside of lambda is a big no-no. It makes code quickly cluttered and is a smell of something not done right. Remember they are supposed to be replacements for boilerplate code rather than structural elements of it. In this case, you can for example use Java8Util.getName(p) or shorthand getName(p) since it's statically imported in the class to avoid try/catch, if it was elsewhere, it is typically good idea to extract such block in own method.